client/src/components/Login.js

```diff
@@ -5,7 +5,7 @@ export default class Login extends Component {
     render() {
         return (
             <div className="container">
-                <form class="form-signin" role="form" action="/login" method="post">
+                <form class="form-signin" action="/login" method="post" style={{maxWidth: "300px"}}>
```

Comment:
Our project is a SPA action "/login" reload page ? you must use handleSubmit here